com.unity.render-pipelines.universal/ShaderLibrary/Unlit.hlsl

```diff
@@ -0,0 +1,50 @@
+
+#ifndef UNLIT_INCLUDED
+#define UNLIT_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Debug/Debugging3D.hlsl"
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl"
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/SurfaceData.hlsl"
+
+half4 UniversalFragmentUnlit(InputData inputData, SurfaceData surfaceData)
+{
+    half3 albedo = surfaceData.albedo;
+
+    #if defined(_ALPHAPREMULTIPLY_ON)
+    albedo *= surfaceData.alpha;
+    #endif
+
+    #if defined(_DEBUG_SHADER)
+    half4 debugColor;
+
+    if(CanDebugOverrideOutputColor(inputData, surfaceData, debugColor))
+    {
+        return debugColor;
+    }
+    #endif
+
+    half4 finalColor = half4(albedo + surfaceData.emission, surfaceData.alpha);
+
+    return finalColor;
+}
+
+// TODO: Legacy code - is it safe to remove this?
+half4 UniversalFragmentUnlit(InputData inputData, half3 color, half alpha)
```

Comment:
Unfortunately, we can't move it to deprecated because that file is used by some really low-level HLSL files (i.e. "ShaderVariablesFunctions.hlsl") and so we'd end up with recursive includes. I guess we'll just have to change the comment.

Reply:
I've added comments before each of these methods informing the user of the deprecation.